### PR TITLE
Add playwright for running UI tests

### PR DESCRIPTION
**Pull request by rossjones**
Adds playwright to the basic prototype so that we can test the prototype where components are built and tested.  Currently only tests the happy path using tribbles.xlsx, but provides the foundation for further testing.  Does not yet test new prototype creation, this will be done as a separate PR

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        node-version: lts/*
+    - name: Install lib dependencies -- todo, check why
+      working-directory: ./lib/importer
+      run: npm i
+    - name: Install dependencies
+      run: npm i
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
```
Comment (simonwo) [inline .github/workflows/playwright.yml:27]: Is it worth using [`--reporter=github`](https://playwright.dev/docs/next/test-reporters#github-actions-annotations) and/or [`--reporter=junit`](https://playwright.dev/docs/next/test-reporters#junit-reporter)?

**Review by simonwo — COMMENTED** [review]
The Github Upload artifact action says:
> Warning: No files were found with the provided path: ./test-results/. No artifacts will be uploaded.

Is it worth fixing this?

**Comment by rossjones** [comment]
> The Github Upload artifact action says:
> 
> > Warning: No files were found with the provided path: ./test-results/. No artifacts will be uploaded.
> 
> Is it worth fixing this?

The files in test-results are only generated on a failure. So should only work when it fails.

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (rossjones) [reply .github/workflows/playwright.yml:27]: Trying it out and if I can get junit to write to test-results as well can fix two things at once.

**Review by simonwo — APPROVED** [review]

**Pull request merged**